tests/validation/tests/v3_api/common.py

```diff
@@ -1603,23 +1619,40 @@ def validate_catalog_app(proj_client, app, external_id, answer=None):
         "the version of the app is not correct"
     # check if associated workloads are active
     ns = app.targetNamespace
-    pramaters = external_id.split('&')
-    assert len(pramaters) > 1, \
-        "Incorrect list of paramaters from catalog external ID"
-    chart = pramaters[len(pramaters)-2].split("=")[1] + "-" + \
-            pramaters[len(pramaters)-1].split("=")[1]
+    parameters = external_id.split('&')
+    assert len(parameters) > 1, \
+        "Incorrect list of parameters from catalog external ID"
+    chart = parameters[len(parameters)-2].split("=")[1] + "-" + \
+            parameters[len(parameters)-1].split("=")[1]
+    app_name = parameters[len(parameters)-2].split("=")[1]
     workloads = proj_client.list_workload(namespaceId=ns).data
     for wl in workloads:
         assert wl.state == "active"
-        assert wl.workloadLabels.chart == chart, \
-            "the chart version is wrong"
+        chart_deployed = get_chart_info(wl.workloadLabels)
```

Comment:
when get_chart_info() returns the last scenario result `''` what is expected?

Reply:
If chart of helm.sh/chart not available, empty is returned and version check is skipped.